lib/team.rb

```diff
@@ -0,0 +1,17 @@
+class Team
+  attr_reader :team_id,
+              :franchiseId,
+              :teamName,
+              :abbreviation,
+              :stadium,
+              :link
+
+  def initialize(data)
+    @team_id      = data[:team_id].to_i
+    @franchiseId  = data[:franchiseId].to_i
+    @teamName     = data[:teamName].to_s
```

Comment:
I think the `to_s` here is redundant since the CSV::ROW defaults to having the values as strings.

Reply:
I will change it, just being double sure